tests/fixtures/resources/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+services:
+  caikit:
+    image: quay.io/opendatahub/caikit-tgis-serving:fast
+    platform: linux/amd64 # allows running this image on apple silicon
+    volumes:
+      - ./flan-t5-small-caikit/:/mnt/models/ # downloaded when running tests
+      - ./caikit-tgis.yml:/caikit/config/caikit.yml
+    ports:
+      - 8085:8085 # grpc
+      - 8080:8080 # http
+  tgis:
+    image: quay.io/opendatahub/text-generation-inference:stable
```

Comment:
Is there a reason why we use different tags for the images? Why not use either `stable`` of `fast` for both ?
The two images are: 
* `quay.io/opendatahub/caikit-tgis-serving:fast`
* `quay.io/opendatahub/text-generation-inference:stable` ?

Reply:
The problem  was that the `stable` image for `caikit-tgis-serving` was not (yet) updated. I just checked and it has been updated a few days ago. Will update.